React/Modules/RCTDevLoadingView.m

```diff
@@ -57,59 +64,64 @@ - (void)setBridge:(RCTBridge *)bridge
 
 - (void)showWithURL:(NSURL *)URL
 {
-  dispatch_async(dispatch_get_main_queue(), ^{
-
-    _showDate = [NSDate date];
-    if (!_window && !RCTRunningInTestEnvironment()) {
-      CGFloat screenWidth = [UIScreen mainScreen].bounds.size.width;
-      _window = [[UIWindow alloc] initWithFrame:CGRectMake(0, 0, screenWidth, 22)];
-      _window.backgroundColor = [UIColor blackColor];
-      _window.windowLevel = UIWindowLevelStatusBar + 1;
-
-      _label = [[UILabel alloc] initWithFrame:_window.bounds];
-      _label.font = [UIFont systemFontOfSize:12.0];
-      _label.textColor = [UIColor grayColor];
-      _label.textAlignment = NSTextAlignmentCenter;
-
-      [_window addSubview:_label];
-      [_window makeKeyAndVisible];
-    }
-
-    NSString *source;
-    if (URL.fileURL) {
-      source = @"pre-bundled file";
-    } else {
-      source = [NSString stringWithFormat:@"%@:%@", URL.host, URL.port];
-    }
-
-    _label.text = [NSString stringWithFormat:@"Loading from %@...", source];
-    _window.hidden = NO;
-
-  });
+  if (isEnabled) {
```

Comment:
There's no style guide but avoiding multiple exits sometimes leads to extra convoluted code. When you return early it can make your code simpler because you're reducing the state space of your program and keeping the code visually easier to read because there's less indentation.